spring-cloud-aws-messaging/src/main/java/org/springframework/cloud/aws/messaging/core/QueueMessageChannel.java

```diff
@@ -41,9 +45,8 @@
  */
 public class QueueMessageChannel extends AbstractMessageChannel implements PollableChannel {
 
-	static final String MESSAGE_RECEIVING_ATTRIBUTE_NAMES = "All";
-	private static final String RECEIPT_HANDLE_MESSAGE_ATTRIBUTE_NAME = "ReceiptHandle";
-	private static final String MESSAGE_ID_MESSAGE_ATTRIBUTE_NAME = "MessageId";
+	static final String ATTRIBUTE_NAMES = "All";
```

Comment:
Because on a SQS message there are attributes and message names (two different things). It's only a "coincidence" that the constant value is the same. But the name of the constants is confusing, I'm going to change it.